src/components/Input/inputwithlabel.style.css

```diff
@@ -1,39 +1,29 @@
 .input {
     display: flex;
     flex-direction: column;
-    position: relative;
-    padding-bottom: 2.6rem;
+    gap:.5rem;
 
   }
   .label {
     color:white;
     font-size: 1.1rem;
-    position: absolute;
-    top: -2rem;
     font-family: sans-serif;;
   }
   
-  ::placeholder {
+  .input::placeholder {
     font-size: 0.9rem;
     font-family: sans-serif;
   }
   
-  input,
-  textarea {
+ .inputBar {
     padding: 0.7rem;
     padding-left: 1rem;
-    width: 18rem;
-    height: 1.5rem; 
     border-radius: 0.5rem;
     margin-top: 0.2rem;
     color: white; 
     background-color: #222; 
     border: 1px solid black; 
```

Comment:
the `background-color` and `border-color` are not the same as the Figma design. You don't have to guess, just click on the component and copy its style and colors from Figma.
![image](https://github.com/ASU-Semicolon/The-New-Dashboard/assets/13494749/847da9fa-7dd3-4725-b117-d955c714e243)
![image](https://github.com/ASU-Semicolon/The-New-Dashboard/assets/13494749/7a6edfa2-81ac-4f06-a2ce-df0a161caf0f)